package.json

```diff
@@ -128,7 +128,7 @@
     "twilio": "^2.5.1",
     "ua-parser": "^0.3.5",
     "ua-parser-js": "^0.7.12",
-    "uglify-js": "^2.8.22",
+    "uglify-js": "^3.0.28",
```

Comment:
Can we try 2.8.29 first? In [the docs](https://www.npmjs.com/package/uglify-js) it says
```
uglify-js@3 has a simplified API and CLI that is not backwards compatible with uglify-js@2.
```